app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,39 @@
+class PasswordsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  def new
+  end
+
+  def verify 
```

Comment:
@JonathanSR should we make this index or show so that these routes are restful? Or is this built into the gem?